deploy.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
-userAgent="GitHub Autodeploy Bot/1.0.0 (${LP_UA_EMAIL})"
-wikiBaseUrl='https://liquipedia.net/'
-luaFiles=$1
+userAgent="GitHub Autodeploy Bot/1.0.0 (${WIKI_UA_EMAIL})"
```

Comment:
Should the version be bumped @FO-nTTaX ?

Reply:
sure

Reply:
```suggestion
userAgent="GitHub Autodeploy Bot/1.1.0 (${WIKI_UA_EMAIL})"
```